mscore/palette.cpp

```diff
@@ -1271,9 +1271,9 @@ bool Palette::event(QEvent* ev)
 void Palette::write(XmlWriter& xml) const
       {
       xml.stag(QString("Palette name=\"%1\"").arg(XmlWriter::xmlString(_name)));
-      xml.tag("gridWidth", hgrid);
-      xml.tag("gridHeight", vgrid);
-      xml.tag("mag", extraMag);
+      xml.tag("gridWidth", hgrid / guiScaling);
+      xml.tag("gridHeight", vgrid / guiScaling);
+      xml.tag("mag", extraMag / guiScaling);
```

Comment:
If possible, it would probably be better to apply scaling when painting rather than multiplying and dividing parameters on reading and writing them.

Reply:
That's how I do it for the QML palettes indeed, but I didn't see a way to do that back with the old-style palettes when I originally implemented this several years back.  I will look again, but FWIW this code is probably not normally ever executed.  These old-style palettes may never get read or written, they are used only for things like the key signature, time signature, and drum palettes, which are constructed on the fly AFAIK,